app/scripts/lib/snap-keyring/utils/isBlockedUrl.test.ts

```diff
@@ -10,7 +10,7 @@ describe('isBlockedUrl', () => {
     allowedEvents: [],
   });
   const phishingController = new PhishingController({
-    // @ts-expect-error TODO: Resolve/patch mismatch between messenger types
+    // @ts-expect-error TODO: Resolve mismatch between base-controller versions.
```

Comment:
Note: `@metamask/phishing-controller` is now on `@metamask/base-controller@7.0.0`, while the rest of extension is using `@metamask/base-controller@6.0.2`.

Reply:
Yes exactly